I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/capitulo_1/ChegadaBienvenidoHostel.rpy", line 17, in script
    recepcionista "Siga-me, irei levá-la para a sua cama."
Exception: Sayer 'recepcionista' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/capitulo_1/ChegadaBienvenidoHostel.rpy", line 17, in script
    recepcionista "Siga-me, irei levá-la para a sua cama."
  File "renpy/ast.py", line 681, in execute
    who = eval_who(self.who, self.who_fast)
  File "renpy/ast.py", line 574, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'recepcionista' is not defined.

Windows-10-10.0.19041
Ren'Py 7.4.8.1895
Mochilona 1.0
Tue Nov  9 22:08:41 2021
